### ISSUE TITLE
Layer not showing with symbology based on columns with number formats

### ISSUE THREAD

**Issue by corgok** [tomchadwin/qgis2web#695]
Hi Tom,
when I use columns that are formatted as integer or float for vector symbology, the layers will not show. If I write same values into text columns, they are showing. For graduated symbols I can only use columns with number formats in QGIS.
Best
Cordula

**Comment by tomchadwin** [comment]
I can't recreate that here. Can you give a few more details:

- QGIS version
- qgis2web version
- OS

The easiest way to make progerss is if you can upload your layer so we can try to recreate the problem.

**Comment by corgok** [comment]
I use QGIS 2.18.14. I downloaded qgis2web last week, it's 2.34.0
OS windows 10
In the file, "soemil" is the original column that did not work(neither VandomrID), and it works with "stringSoem". 

[VandomrSimplified.zip](https://github.com/tomchadwin/qgis2web/files/2548514/VandomrSimplified.zip)

**Comment by tomchadwin** [comment]
Thanks for the details and the uploads. I still cannot get it to fail. I suspect this is because I don't know how you are styling the layer. Can you either export and upload the failing layer's style, or upload your QGIS project file?

**Comment by corgok** [comment]
Today I cannot reproduce it neither, so from my side the problem seems to be solved. On the day of installation, I tried several layers and had problems with them all. My best guess is now, that the restart in between settled the issue. I attached output from the first day. Besides the attached example, I have one (unfortunately too big for upload) where the "other values" are active  and the whole layer gets that colour. It shows that the values somehow do not get recognized and end unclassified. I use point as decimal separator but it is not default in Denmark. My initial thought was that somewhere in the system comma is used, but the values in the layers appear to be fine.
[qgis2web_2018_11_02-14_38_19_413000.zip](https://github.com/tomchadwin/qgis2web/files/2552367/qgis2web_2018_11_02-14_38_19_413000.zip)

**Comment by corgok** [comment]
The blank map is exactly the problem. It has two version of the layer active that I uploaded earlier. When I open it, I can also see the legend entries for both, but the layers are invisible. It was an automatic generation without editing. I tried several times with different layers. I do not know if it is worth to dig deeper. Today I can produce the output that I want. If the problem reoccurs I will update you. Thanks for your effort!
